notebooks/0.download-data/nbconverted/0.download-data.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Module 0: Data Download
+#
+# In this notebook, we will download two datasets generated by the [MitoCheck Consortium](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3108885/).
+#
+# Below is a summarized description of the dataset. For more details, please visit [this link](https://zenodo.org/records/7967386):
+#
+# This dataset contains supplementary files from the mitocheck_data repository, focusing on extracted and normalized single-cell features that were too large to upload to GitHub [repo](https://github.com/WayScience/mitocheck_data).
+#
+# The data includes nuclei features from various cell subsets:
+#
+# - **Negative Control**: Features from cells transfected with scrambled siRNA.
+# - **Positive Control**: Features from cells transfected with siRNA targeting mitosis-related genes.
+# - **Training Data**: Features from cells manually labeled with phenotypic classes by the MitoCheck Consortium.
+#
+# Additionally, the dataset includes:
+#
+# - **IDR Stream Extracted Features**: Single-cell features extracted using CellProfiler and DeepProfiler.
+# - **Normalized Data**: Features normalized using a StandardScaler derived from negative control data.
+#
+# The data relates to the MitoCheck Consortium's original screen, hosted on the Image Data Resource (IDR) under accession [idr0013](https://idr.openmicroscopy.org/webclient/?show=screen-1101) (screenA).
+
+# In[1]:
+
+
+import pathlib
+import zipfile
+
+import requests
+
+# ## Helper functions
+#
+# Funtions that are used in this notebook for downloading the dataset.
+
+# In[2]:
+
+
+def download_and_extract(url, extract_to):
```

Comment:
consider type hints? Also `extract_to` is a somewhat vague var name. Consider changing